### PR TITLE
[Doc Enhancement][Messaging extensions][2189559]

### PR DESCRIPTION
**Pull request by v-silakshmi**

### CODE REVIEW

File: msteams-platform/messaging-extensions/what-are-messaging-extensions.md [hunk 1]
```diff
@@ -45,6 +45,81 @@ Action commands are used to present the users with a modal popup to collect or d
 The action commands are triggered from the compose message area, the command box, or from a message. When the command is invoked from a message, the initial JSON payload sent to your bot includes the entire message it was invoked from. The following image displays the messaging extension action command task module:
 ![messaging extension action command task module](~/assets/images/task-module.png)
 
+#### Code snippets for action commands
```
Comment (v-habibaumme) [inline msteams-platform/messaging-extensions/what-are-messaging-extensions.md:48]: Make this heading as second level before to the code sample section

File: msteams-platform/messaging-extensions/what-are-messaging-extensions.md [hunk 2]
```diff
@@ -54,6 +129,84 @@ The following image displays the messaging extension search command task module:
 
 ![messaging extension search command](~/assets/images/search-extension.png)
 
+#### Code snippets for search commands
```
Comment (v-habibaumme) [inline msteams-platform/messaging-extensions/what-are-messaging-extensions.md:132]: Make this heading as second level before to the code sample section and remove for search commands from the heading to follow the consistency.

Comment (v-silakshmi) [reply msteams-platform/messaging-extensions/what-are-messaging-extensions.md]: Made changes as advised

**Review by v-habibaumme — APPROVED** [review]
Approved

File: msteams-platform/messaging-extensions/what-are-messaging-extensions.md [hunk 3]
```diff
@@ -66,6 +66,158 @@ The following images display link unfurling feature when a link is pasted in mes
 
 ![link unfurling](../assets/images/messaging-extension/link-unfurl.gif)
 
+## Code snippets
+
+The following code provides an example of action based for messaging extensions:
+
+# [Node.js](#tab/nodejs)
```
Comment (Wajeed-msft) [inline msteams-platform/messaging-extensions/what-are-messaging-extensions.md:73]: To keep it consistent with other doc pages, could you please change the sequence to bring C# tab first? 
image

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: msteams-platform/messaging-extensions/what-are-messaging-extensions.md (hunk 3 above)
Comment (v-silakshmi) [reply msteams-platform/messaging-extensions/what-are-messaging-extensions.md]: made changes as advised.

**Review by v-habibaumme — APPROVED** [review]
Approved

**Review by Wajeed-msft — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**